cli/cmd/runner.go

```diff
@@ -183,4 +183,6 @@ type runnerArgs struct {
 
 	apiPostBody string
 	apiPutBody  string
+
+	channelInspectOutputFormat string
```

Comment:
How does this work? I don't see this variable used anywhere.

Reply:
It is used in the `channel inspect` command